### PR TITLE
Created more complex shortestPath test case to reproduce limitation

### PR DESCRIPTION
**Pull request by craigtaverner**

### CODE REVIEW

File: community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/ShortestPathLongerAcceptanceTest.scala [hunk 1]
```diff
@@ -0,0 +1,81 @@
+package org.neo4j.internal.cypher.acceptance
```
Comment (pontusmelke) [inline community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/ShortestPathLongerAcceptanceTest.scala:20]: license

**Comment by cleishm** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;6P5K5kXB&#x2F;494-mergepattern-getorcreate-operator"> MergePattern (GetOrCreate operator)</a>

**Comment by Mats-SX** [comment]
This needs a rebase.

**Comment by henriknyman** [comment]
This is already merged as part of this PR https://github.com/neo4j/neo4j/pull/6004

**Pull request closed without merging**